.github/workflows/build_and_deploy.yml

```diff
@@ -137,3 +120,55 @@ jobs:
           SLACK_TITLE: Failure deploying release to test
           SLACK_MESSAGE: Failure deploying release to test - Docker tag ${{ needs.build.outputs.docker-image-tag }}
           SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_URL }}
+
+  deploy_production:
```

Comment:
This is the same code as above. Can we use a [matrix](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs) instead? With `max-parallel: 1` to make it sequential.